NOpenType/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace NRasterizer
+{
+    internal static class Constants
+    {
+        public const short EmSquareSize = 2048;
+
+        public const int PointsPerInch = 72;
+
+        internal const int FontToPixelDivisor = EmSquareSize * PointsPerInch;
+    }
+}
```

Comment:
The risk with a `Constants` class is all kinds of weird things gets dumped here. Instead, lets have constants private in classes where needed. Perhaps even local to the function! The `PointsPerInch` does not feel like a constant either in the same way the EmSquare size is. I liked the EmSquare class with a Size constant. :-) But it could also go inside the `Typeface` class I guess?